src/wix/test/WixToolsetTest.BuildTasks/WixToolsetTest.BuildTasks.csproj

```diff
@@ -22,6 +22,12 @@
     <ProjectReference Include="..\..\WixToolset.Core.TestPackage\WixToolset.Core.TestPackage.csproj" />
   </ItemGroup>
 
+  <!-- Copy the x64 binaries for unittests since we've standardized on VS2022 (which is 64-bit only) -->
+  <ItemGroup>
+    <None Include="..\..\wixnative\x64\mergemod.dll" CopyToOutputDirectory="PreserveNewest" />
+    <None Include="$(BaseOutputPath)$(Configuration)\x64\wixnative.exe" CopyToOutputDirectory="PreserveNewest" />
+  </ItemGroup>
+
```

Comment:
It seems these can go into the new `src\wix\test\Directory.Build.props` instead of in each project.